Frontend/ui-driver/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -176,7 +175,25 @@ config =
                 empty_referral_auto : "ny_ic_refer_now_auto_ny_green,https://assets.moving.tech/beckn/common/driver/images/ny_ic_refer_now_auto_ny_green.png",
                 empty_referral_cab : "ny_ic_refer_now_cab_ny,https://assets.moving.tech/beckn/common/driver/images/ny_ic_refer_now_cab_ny.png"
               },
-              enableHvSdk : true
+              enableHvSdk : true,
+              purpleRideConfig : [ {
```

Comment:
@Shikhar-24 Instead of array can we have object ?

Reply:
done @kavya-shree-s